tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -2430,13 +2430,16 @@ public function testAlterTableForeignKeyConstraintValidationNotRunningWithDisabl
     public function testLiteralSupport()
     {
         $createQuery = <<<'INPUT'
-CREATE TABLE `test` (`real_col` DECIMAL)
+CREATE TABLE `test` (`real_col` FOO)
 INPUT;
         $this->adapter->execute($createQuery);
         $table = new Table('test', [], $this->adapter);
         $columns = $table->getColumns();
         $this->assertCount(1, $columns);
-        $this->assertEquals(Literal::from('decimal'), array_pop($columns)->getType());
```

Comment:
`DECIMAL` is an accepted type, so `->getType()` was returning `"decimal"`, but since we were doing `assertEquals`, which equates to `Literal::from('decimal') == 'decimal'`, the test was passing (as `Literal` was being converted to a string).

This is also why I added an assertion on the actual type of `getType()`.